plugins/immer/src/index.ts

```diff
@@ -0,0 +1,33 @@
+import { PluginCreator, RootReducers, Models } from '@rematch/core'
+import { combineReducers } from 'redux'
+import produce from 'immer'
+
+function combineReducersWithImmer(reducers: RootReducers) {
```

Comment:
change type to `ReducersMapObject`

Reply:
ReducersMapObject don't export. How do I use?

Reply:
Try:

```js
import { ReducersMapObject } from 'redux'
```

Reply:
OK, that is worked. I committed.